docs/build.md

```diff
@@ -280,6 +280,28 @@ The following compilation options are also available to tweak performance (yes,
 
 **Windows**
 
+#### w64devkit
```

Comment:
since you think `w64devkit` is better I moved it above.

I think its useful to have another example, I had most of this already installed.